test/e2e/models/CodeServer.ts

```diff
@@ -88,6 +88,13 @@ export class CodeServer {
       }),
       "utf8",
     )
+
+    const extensionsDir = path.join(__dirname, "../extensions")
+
+    await fs.writeFile(
```

Comment:
Should we leave a comment here saying this is a workaround until we fix the file not generating automatically?

Reply:
Yes! Otherwise, we will surely forget 😂